manifests/i/IObit/DriverBooster/10.3.0.124/IObit.DriverBooster.locale.en-US.yaml

```
@@ -1,25 +1,24 @@
-# Created using wingetcreate 1.1.2.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
-
-PackageIdentifier: IObit.DriverBooster
-PackageVersion: 10.2.0
-PackageLocale: en-US
-Publisher: IObit
-PublisherUrl: https://www.iobit.com/en/index.php
-PublisherSupportUrl: https://www.iobit.com/en/onlinefeedback.php
-PrivacyUrl: https://www.iobit.com/en/privacy.php
-Author: IObit
-PackageName: Driver Booster
-PackageUrl: https://www.iobit.com/en/driver-booster.php
-License: Proprietary
-LicenseUrl: https://www.iobit.com/en/eula.php
-Copyright: © 2005 - 2021 IObit. All Rights Reserved
-CopyrightUrl: https://www.iobit.com/en/eula.php
-ShortDescription: All-in-one PC optimizer to clean junk files, sweep privacy traces, and boost startup. Speed up with new AI Mode. Smarter & Faster than ever before.
-Description: All-in-one PC optimizer to clean junk files, sweep privacy traces, and boost startup. Speed up with new AI Mode. Smarter & Faster than ever before.
-Moniker: driver-booster
-Tags:
-- driver-booster
-ManifestType: defaultLocale
-ManifestVersion: 1.2.0
-
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: IObit.DriverBooster
+PackageVersion: 10.3.0.124
```

```suggestion
PackageVersion: 10.3.0
```
This should match control panel, or it will cause an upgrade loop. The version folder will need to be renamed